crates/rethnet_eth/src/remote/serde_with_helpers.rs

```diff
@@ -1,7 +1,7 @@
 /// helper utilities for use with serde's serialize_with and deserialize_with
 use std::fmt::Write;
 
-use crate::U256;
+use ruint::Uint;
```

Comment:
```suggestion
use revm_primitives::ruint::Uint;
```

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/4218/commits/6d1e0ba16866a04340148603d71fd79c6a95023b